src/swell/utilities/jinja2.py

```diff
@@ -7,28 +7,75 @@
 # --------------------------------------------------------------------------------------------------
 
 
-import jinja2
+import jinja2 as j2
 
 
```

Comment:
To allow type hints on the `template_string_jinja2` function below.

```suggestion
from swell.utilties.logger import Logger

```